ui/qt-app/Glue.hs

```diff
@@ -89,6 +89,8 @@ knitFaceToUI UiFace{..} KnitFace{..} =
           return ()
       }
 
+    justOptNumber key = maybe [] (\value -> [Knit.ArgKw key . Knit.ExprLit . Knit.toLit . Knit.LitNumber $ fromIntegral value])
```

Comment:
Could you please write type of this function? It's not obvious (I mean, it took me some time to figure it out).

Reply:
It's just copy-paste from vty glue :man_shrugging: 
Will add.